codec-http/src/main/java/io/netty/handler/codec/http/cors/CorsHandler.java

```diff
@@ -66,7 +66,7 @@ public void channelRead(final ChannelHandlerContext ctx, final Object msg) throw
     }
 
     private void handlePreflight(final ChannelHandlerContext ctx, final HttpRequest request) {
-        final HttpResponse response = new DefaultFullHttpResponse(request.protocolVersion(), OK, true, true);
+        final HttpResponse response = new DefaultFullHttpResponse(request.protocolVersion(), OK, true, false);
```

Comment:
The logic was flipped so that `DefaultHeaders` was not on the `else` branch condition.  Was just experimenting and don't think this is necessary...I will revert.